jaxb-adapters/src/main/java/org/hawkular/accounts/jaxb/adapters/ZonedDateTimeXmlAdapter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.accounts.jaxb.adapters;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+public class ZonedDateTimeXmlAdapter extends XmlAdapter<String, ZonedDateTime> {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ISO_DATE_TIME;
+
+    @Override
+    public ZonedDateTime unmarshal(String s) throws Exception {
+        if (null == s || s.isEmpty()) {
+            return null;
+        }
+
+        return ZonedDateTime.parse(s, formatter);
+    }
+
+    @Override
+    public String marshal(ZonedDateTime zonedDateTime) throws Exception {
+        if (null == zonedDateTime) {
+            return null;
+        }
+
+        return zonedDateTime.withZoneSameInstant(ZoneOffset.UTC).format(formatter);
```

Comment:
> client can easily manipulate

Well, "easily" probably depends on the complexity of the entities returned by the server, which is probably always very small in case of accounts. In Invetory or Metrics, where the server can return a long and deep tree of various entities, /me the client software developer would probably not find it easy to reliably find all dates in the tree and transform them. That's just a side note. The PR is OK for the present stage of the project.